### PR TITLE
BSA 36 - Avoid error message appears in any case before user write something

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/BSA-36

**Blocked by**

https://github.com/oat-sa/tao-item-runner-qti-fe/pull/63

**Other necessary PR to be merged for seeing complete feature working**

`Pending to add link`

**Description**

Avoid error message appears in any case before the user write something

**AC**

- [x] Message error only appears after user write something **AND** don't fit pattern mask

### CODE REVIEW

File: views/package-lock.json [hunk 1]
```diff
@@ -10,9 +10,8 @@
       "integrity": "sha512-Z/fRV8TSxitZwO40zeL8T+pLbxoqp5L2dDqWf+0kPOV3Guf2juP1eXZygiXhwmHHOTgwpSeKSC0sOJJATsL5xQ=="
     },
     "@oat-sa/tao-item-runner-qti": {
-      "version": "0.4.19",
-      "resolved": "https://registry.npmjs.org/@oat-sa/tao-item-runner-qti/-/tao-item-runner-qti-0.4.19.tgz",
-      "integrity": "sha512-scjmtp1LEZyyhzwz5LKjZ/0BO47cC3KjWrzoZGfaAKZW3bn8zJpbTCI5aDQDOD8A3FPBfHY0IFs87jsHyG4CWA=="
+      "version": "github:oat-sa/tao-item-runner-qti-fe#ca1ac75dfe0377ab9b922b52bcbdf63d524ca3be",
```
Comment (bziondik) [inline views/package-lock.json:13]: Not right, should be npm version
`rm -rf node_modules/`
`npm install`

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (bziondik) [inline package-lock.json:1]: Remove this file

File: views/package-lock.json (hunk 1 above)
Comment (wazelin) [reply views/package-lock.json:13]: `npm update` I believe, `install` will use the lock file's data.

File: package-lock.json (hunk 2 above)
Comment (wazelin) [reply package-lock.json:1]: Still not removed, @habemuscode.

Comment (habemuscode) [reply package-lock.json:1]: Ok, I understand to remove before `npm install` and will be generated again

Comment (habemuscode) [reply package-lock.json]: I see, lock outside views/

**Pull request merged**